wouso/resources/templates/cpanel/races/all.html

```diff
@@ -205,6 +205,27 @@ <h3 class='label-3'><span class="label label-danger label-sm">{% trans 'no' %}</
             $(this).data('show', true);
         }
     });
+
+    $(document).on("click", ".race_can_play_toggle", function() {
+        var race_pk = $(this).data('race_pk');
+        var button = $(this);
+
+        $.ajax({
```

Comment:
@catalin-oancea, the request should be PUT - you are not requesting data (what a GET request implies), you are updating a specific element.